src/components/network-chart/NetworkChartRow.js

```diff
@@ -305,12 +305,17 @@ class NetworkChartRowBar extends React.PureComponent<NetworkChartRowBarProps> {
 type NetworkChartRowProps = {|
   +index: number,
   +marker: Marker,
+  +markerIndex: MarkerIndex,
   // Pass the payload in as well, since our types can't express a Marker with
   // a specific payload.
   +networkPayload: NetworkPayload,
   +timeRange: StartEndRange,
   +width: CssPixels,
   +threadIndex: ThreadIndex,
+  +isRightClicked: boolean,
+  +onLeftClick?: MarkerIndex => mixed,
+  +onRightClick?: MarkerIndex => mixed,
+  +shouldDisplayTooltips: () => boolean,
```

Comment:
This is a bit weird to pass such a function.
Another solution would be to make the tooltip shown by the parent, but I thought it was also a bit more painful to implement (needing to pass the mouse position especially).